gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/sql/execute/MemberStatisticsMessage.java

```diff
@@ -179,6 +184,21 @@ private String getUserDir(){
     return System.getProperty("user.dir");
   }
 
+  private String getLogFile() {
+    Logger rootLogger = Logger.getRootLogger();
+    Appender appender;
+    if (rootLogger != null) {
+      Enumeration<?> e = rootLogger.getAllAppenders();
+      while (e.hasMoreElements()) {
+        appender = (Appender)e.nextElement();
+        if (appender instanceof FileAppender) {
+          return ((FileAppender)appender).getFile();
+        }
+      }
+    }
+    return "";
+  }
```

Comment:
Removed getUserDir and getLogFile methods from MemberLogMessage. Setting them in instance variables while creating new MemberLogMessage object.